pkg/generate/app/cmd/newapp.go

```diff
@@ -710,20 +711,150 @@ func (c *AppConfig) Run() (*AppResult, error) {
 	}, nil
 }
 
+// followRefToDockerImage follows a buildconfig...To/From reference until it t
+// erminates in docker image information. This can include dereferencing chains
```

Comment:
erminate (verb) - meaning to improperly line break comments in code. 

fixed :)